lms/djangoapps/certificates/models.py

```diff
@@ -97,6 +97,7 @@ class CertificateWhitelist(models.Model):
     """
     class Meta:
         app_label = "certificates"
+        unique_together = [['course_id', 'user']]
```

Comment:
I was going to comment a style nit that these should be tuples (because that's usually what we use) then I realized the Django docs uses lists. Sigh. So I guess ignore this comment :)